install_files/ansible-base/securedrop-staging.yml

```diff
@@ -1,10 +1,22 @@
 #!/usr/bin/env ansible-playbook
 ---
-  # Staging machines should be provisioned with local code,
-  # rather than pulling down stable releases from the app code.
-  # Therefore build the local debs before provisioning the staging hosts.
-- include: build-deb-pkgs.yml
-  tags: build_debs
+- name: Scrape build directory
+  hosts: localhost
+  tasks:
+    - name: Establish list of files in build/
+      find:
+        paths: "../../build"
+      register: search_deb_results
+
+    - set_fact:
+        expected_debs: "{{hostvars['app-staging']['local_deb_packages'] + hostvars['mon-staging']['local_deb_packages']}}"
+        found_debs: "{{ search_deb_results.files | map(attribute='path') | list }}"
+
+    - name: Fail if package is missing from build dir
+      fail:
+        msg: "run 'make build-debs' - ../../build/{{item}} missing"
+      when: "'../../build/'+item not in found_debs"
+      with_items: "{{ expected_debs }}"
```

Comment:
Nit: It would be slightly more readable to use a `|basename` filter on the `found_debs` vars declaration, then the `when` conditional wouldn't need to prefix the build path.